src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -0,0 +1,436 @@
+      return chatInlineAlert.isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isMessageOnChat(String message) {
+    try {
+      WebElement userPostedMessage = userPostedMessage(message);
+      wait.forElementVisible(userPostedMessage);
+      return userPostedMessage.isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isUserWelcomeMessageDisplayed(String userName) {
+    try {
+      wait.forElementVisible(userWelcomeMessage(userName));
+      return userWelcomeMessage(userName).isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateMessageHeaderDispayed() {
+    try {
+      wait.forElementVisible(privateMessagesHeader);
+      return privateMessagesHeader.isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isPermissionsErrorTitleDisplayed() {
+    try {
+      wait.forElementVisible(permissionsErrorTitle);
+      return permissionsErrorTitle.isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateMessageNotificationDisplayed() {
+    try {
+      wait.forElementVisible(privateMessageNotification);
+      return privateMessageNotification.isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateNotificationCountDisplayed(int notificationCount) {
+    try {
+      wait.forElementVisible(userPrivateMessageNotification(notificationCount));
+      return userPrivateMessageNotification(notificationCount).isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateMessageButtonDisplayed() {
+    try {
+      return privateMassageButton.isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateChatOpen() {
+    try {
+      wait.forElementVisible(privateChatHeader);
+      return privateChatHeader.isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isRegularUserDropdownDisplayed() {
+    if (userDropDownActionsElements.size() == REGULAR_USER_DROPDOWN_ELEMENTS) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  public boolean isUserInPrivateSectionDisplayed(String userName) {
+    try {
+      return userOnChat(userName, PRIVATE_MESSAGE_USER_SELECTOR).isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isChatUnbanMessageDispalyed() {
+    try {
+      wait.forElementVisible(getUserUnbanMessage());
+      return getUserUnbanMessage().isDisplayed();
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public boolean isMessageWallOpened(String userName) {
+    try {
+      Assertion.assertEquals(userName, userNameTitle.getText());
+      return userPageMessageWallTab.isDisplayed();
+    } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)